src/utils/github/getAuthors.ts

```diff
@@ -26,6 +28,137 @@ export async function getAuthors(
     });
   } catch (err) {
     // eslint-disable-next-line no-console
+    // @ts-ignore
+    console.error(err);
+    return [];
+  }
+}
+
+interface RevertAuthorInfo {
+  login: string | null;
+  email: string | null;
+}
+
+function extractRevertedCommitHash(commitMessage: string): string | null {
+  const match = commitMessage.match(/This reverts commit ([a-f0-9]+)/i);
+  return match ? match[1] : null;
+}
+
+async function getRevertCommitDetails(
+  revertedHash: string,
+  owner: string,
+  repo: string,
+  octokitInstance: Octokit
+): Promise<RevertAuthorInfo | null> {
+  try {
+    const revertedCommitDetailsResponse = await octokitInstance.request(
+      'GET /repos/{owner}/{repo}/commits/{ref}',
+      {
+        owner,
+        repo,
+        ref: revertedHash,
+        headers: { 'X-GitHub-Api-Version': '2022-11-28' },
+      }
+    );
```

Comment:
Would it be possible to utilize the higher level octokit apis as we do elsewhere in the code: https://github.com/getsentry/eng-pipes/blob/d4ba7ed6262f9359dc2ae1ce5c01f13db145c48c/src/utils/github/getAuthors.ts#L9 for consistency?